go-controller/pkg/ovn/base_network_controller.go

```diff
@@ -816,7 +816,16 @@ func (bnc *BaseNetworkController) findMigratablePodIPsForSubnets(subnets []*net.
 		}
 		podAnnotation, err := util.UnmarshalPodAnnotation(liveMigratablePod.Annotations, bnc.GetNetworkName())
 		if err != nil {
-			return nil, err
+			// even though it can be normal to not have an annotation now, live
+			// migration is a sensible process that might be used when draining
+			// nodes on upgrades, so log a warning in every case to have the
+			// information available
+			klog.Warningf("Could not get pod annotation of pod %s/%s for NAD %s: %v",
```

Comment:
Don't we want to filter out just `IsAnnotationNotSetError` errors ?

Reply:
And here as well
https://github.com/jcaamano/ovn-kubernetes/blob/37290d04228709ec9d4972853bed7829ba7665ba/go-controller/pkg/kubevirt/pod.go#L269-L272

Reply:
We can take `IsAnnotationNotSetError` into account when deciding whether we log or not, but you asked explicitly for the log offline so I added it unconditionally.

Reply:
fair enough, let's log it all, if we see it's too noise we can reduce it later on with a new PR.